packages/cli/src/Generate.ts

```diff
@@ -234,8 +234,21 @@ ${chalk.dim('```')}
 ${highlightTS(`\
 import { PrismaClient } from '${importPath}'
 const prisma = new PrismaClient()`)}
-${chalk.dim('```')}${breakingChangesStr}${versionsWarning}`
+${chalk.dim('```')}${
+          prismaClientJSGenerator.options?.dataProxy
+            ? `
+
+To use Prisma Client in edge environments like Cloudflare Workers or Vercel Edge Functions, import it like this:
+${chalk.dim('```')}
+${highlightTS(`\
+import { PrismaClient } from '${importPath}/edge'`)}
+${chalk.dim('```')}
+Data Proxy documentation: ${link('https://www.prisma.io/docs/data-platform/data-proxy')}
```

Comment:
```suggestion
Data Proxy documentation: ${link('https://pris.ly/d/data-proxy')}
```

Reply:
> https://pris.ly/d/data-proxy

Oh, nice, it works now, thank you! When I tried this, it redirected me to https://www.prisma.io/docs/concepts/data-proxy instead of https://www.prisma.io/docs/data-platform/data-proxy and I got 404. I don't know if I did something wrong and tried a slightly different URL, or if it was a glitch/bug in the docs.